.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,28 +4,25 @@ about: Request a new feature
 
 ---
 
-<!--
-DELETE ME
+# Checklist
+<!-- Place an x in the boxes to tick them: [x] -->
 
-Make sure you are using the latest version of AntennaPod. Perhaps the feature you are looking for has already been implemented.
+- [ ] I have used the search function to see if someone else has already submitted the same feature request.
+- [ ] I will only create one feature request per issue.
+- [ ] I will describe the problem with as much detail as possible.
 
-Use the search function to see if someone else has already submitted the same feature request. If there is another request already, please upvote the first post instead of commenting something like "I also want this".
+# System info
 
-To make it easier for us to keep track of requests, please only make one feature request per issue.
-
-Give a brief explanation about the problem that may currently exist and how your requested feature solves this problem.
-
-Try to be as specific as possible. Please not only explain what the feature does, but also how. If your request is about (or includes) changing or extending the UI, describe what the UI would look like and how the user would interact with it.
--->
-
-<!-- READ THIS: The latest version may be different depending on your device. You can find the version in AntennaPod's settings. -->
+<!-- The latest version may be different depending on your device. You can find the version in AntennaPod's settings. -->
 **App version**: x.y.z (state whether from Google Play/F-Droid/Custom built APK)
```

Comment:
```
**Current app version**: x.y.z (Where did you download the app?)
<!-- Your desired feature might already be available in the latest version available for your device.
You can find the version in AntennaPod's settings. -->
```

Reply:
I find it very important to state that the latest version might be different. Otherwise, users just write "latest".